src/VisualStudio/IntegrationTests/CSharp/CSharpInteractiveDemo.cs

```diff
@@ -0,0 +1,106 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using Roslyn.VisualStudio.Test.Utilities;
+using Xunit;
+
+namespace Roslyn.VisualStudio.Integration.UnitTests
+{
+    [Collection(nameof(SharedIntegrationHost))]
+    public class CSharpInteractiveDemo
+    {
+        private IntegrationHost _host;
+        private InteractiveWindow _interactiveWindow;
+
+        public CSharpInteractiveDemo(IntegrationHost host)
+        {
+            _host = host;
+            _host.Initialize();
+
+            _interactiveWindow = _host.CSharpInteractiveWindow;
+
+            _interactiveWindow.Show();
+            _interactiveWindow.Reset();
+        }
+
+        [Fact]
+        public void BclMathCall()
+        {
+            _interactiveWindow.SubmitTextToRepl("Math.Sin(1)");
+            Assert.True(_interactiveWindow.DoesLastReplOutputEqual("0.8414709848078965"));
```

Comment:
Assert.True/False on a method that returns bool is somewhat an antipattern here. We're better off just having _interactiveWindow.LastReplOutput, and call Assert.Equals() directly. That way when the test fails, xunit can give a clear "I got this, we expected this" exception instead of just a "I got false, expected true" error.